docs/configuration-v1_1-exp.md

```diff
@@ -9,16 +9,18 @@ The Fedora CoreOS configuration is a YAML document conforming to the following s
 * **ignition** (object): metadata about the configuration itself.
   * **_config_** (objects): options related to the configuration.
     * **_merge_** (list of objects): a list of the configs to be merged to the current config.
-      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline`.
-      * **_inline_** (string): the contents of the config. Mutually exclusive with `source`.
+      * **_source_** (string): the URL of the config. Supported schemes are `http`, `https`, `s3`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified. Mutually exclusive with `inline` and `local`.
+      * **_inline_** (string): the contents of the config. Mutually exclusive with `source` and `local`.
+      * **_local_** (string): a local path to the contents of the config. Mutually exclusive with `source` and `inline`.
```

Comment:
Might be worth adding a mention to the `-d/--files-dir` requirement in the specification.